app/main/views/briefs.py

```diff
@@ -0,0 +1,41 @@
+from flask import render_template, request, redirect, flash
+from flask_login import current_user
+
+from ..helpers import login_required
+from ..helpers.briefs import (
+    get_brief,
+    check_supplier_is_eligible_for_brief,
+    send_brief_clarification_question,
+)
+from ...main import main
+from ... import data_api_client
+
+
+@main.route('/opportunities/<int:brief_id>/ask-a-question', methods=['GET', 'POST'])
+@login_required
+def ask_brief_clarification_question(brief_id):
+    brief = get_brief(data_api_client, brief_id)
+    check_supplier_is_eligible_for_brief(brief, current_user.supplier_id)
+
+    error_message = None
+    clarification_question_value = None
+
+    if request.method == 'POST':
+        clarification_question = request.form.get('clarification-question', '').strip()
+        if not clarification_question:
+            error_message = "Question cannot be empty"
+        elif len(clarification_question) > 5000:
```

Comment:
Oh, I forgot to make a note about limits. I've mentioned this to Ralph briefly, but I agree we need to decide what we should do.

I think we need an upper length limit to protect buyers from malicious emails. I'm not enforcing a word limit (even though there's a counter on the form) to avoid implementing it again before we're sure there's a need for it. 5000 character limit is what we used for framework clarification questions.